src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public  Duke(){
+
+    }
+    ArrayList<Task> tasks = new ArrayList<>();
+    private final static String horLine = "------------------------------------------------------";
+    private final static String welcomeMsg = horLine + "\nHello! I'm Duke\n" +
```

Comment:
> Constant names must be all uppercase using underscore to separate words.

Perhaps you can name them something more semantic as well, like `HORIZONTAL_LINE` and `WELCOME_MESSAGE`

Reply:
Yes, you are correct. I will do that in my next commit.